src/components/reviews/Reviews.jsx

```diff
@@ -127,10 +127,26 @@ class Reviews extends React.Component {
         }
       };
     } else if (sortOrder === 'relevant') { // not sure how to sort for relevancy?
```

Comment:
Do you want to keep this comment in here? Looks like you came up with a solution